pkg/bookmarks/bookmarks.go

```diff
@@ -72,3 +89,22 @@ func ReadAll(path string) (map[string]Bookmark, error) {
 
 	return results, nil
 }
+
+type ErrNonExistingBookmark string
```

Comment:
Do not use this error definition pattern. Its not being used anywhere in the codebase.